upload/serializers.py

```diff
@@ -67,11 +69,13 @@ def create(self, validated_data):
         flag_names = (
             validated_data.pop("flags") if "flags" in validated_data.keys() else []
         )
-        _ = (
+
+        if "version" in validated_data.keys():
             validated_data.pop("version")
```

Comment:
I suppose I could do something like 
```
validated_data.pop("version", default=None)
```
but I don't think it's clear that `default` is necessary or else this will throw an error if
version is not found

would it be cleaner to do what's below or to do what's above with a comment explaining that default is necessary? 
```
try:
  validated_data.pop("version", default=None)
except KeyError:
  pass
```

Reply:
personally i think `validated_data.pop("version", default=None)` is clear enough. you don't assign the return value to anything so it's just getting rid of the value if it's in there

there's room to disagree there, though. if you want to be more explicit, i think tweak the current approach:
```
if "version" in validated_data:
    validated_data.pop("version")
```

i think the above is more "pythonic" + probably slightly more efficient (not that it matters here). checking for it in `.keys()` is probably O(n), but `key in dict` is probably O(1) haha